### PR TITLE
Shrink image size

### PR DESCRIPTION
**Pull request by LindirQuenya**
This should shrink the image significantly. When I tried building it locally, it cut the image size to 120M.

### CODE REVIEW

**Comment by LindirQuenya** [comment]
I have now added some new changes, which reduce the total image size (incl. snapshot) to ~140M.
Most of that was from removing unneeded kernel modules. I also enabled compression and moved most interfaces to virtio.
Note that this is no longer a drop-in replacement for what is already in Flashpoint; the Apache Webserver arguments line in services.json needs to be changed to:
["-m", "512", "-net", "nic,model=virtio-net-pci", "-net", "user,hostfwd=tcp:127.0.0.1:22500-:80", "-qmp", "tcp:127.0.0.1:4444,server,nowait", "-qmp", "tcp:127.0.0.1:4445,server,nowait", "-drive", "file=alpine.qcow2,if=virtio", "-serial", "stdio", "-loadvm", "quick", "-display", "none"]
This is because of the move to virtio which should (in theory) improve performance. I hope it outweighs the performance decrease caused by enabling compression on the image.

**Comment by LindirQuenya** [comment]
Note that for this to compile/make locally before you merge the changes, you need to change setup.sh line 100 from:
`wget https://raw.githubusercontent.com/FlashpointProject/flashpointvm/master/needed_mods.txt -O /root/needed_mods.txt`
to 
`wget https://raw.githubusercontent.com/LindirQuenya/flashpointvm/master/needed_mods.txt -O /root/needed_mods.txt`

File: setup.sh [hunk 1]
```diff
@@ -78,7 +78,49 @@ rc-update add gamezip default
...
+    # If it's not on the list, remove it, and echo a nice message.
+    echo "Deleting module: $i"
+    rm "$i"
+  fi
+done
+
+# build tools aren't needed anymore, remove them.
+apk del build-base fuse-dev git
+# Other components that we don't need.
+apk del php-openssl php-pdo_sqlite
```
Comment (XXLuigiMario) [inline setup.sh:119]: These components are actually needed by some specific games. The PHP packages installed at the beggining of the script are all used in some capacity.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# flashpointvm
```
Comment (XXLuigiMario) [inline README.md:1]: README looks good, but I'd prefer if it were moved to a separate PR.

**Review by XXLuigiMario — CHANGES_REQUESTED** [review]
Nice work! After some use on my copy of Flashpoint, I didn't notice any issues. There are just a couple minor things that need to be addressed. Once that's taken care of I'll merge this ASAP.

Sorry for not looking at this earlier, this month has been quite busy.

File: setup.sh (hunk 1 above)
Comment (LindirQuenya) [reply setup.sh]: Oh, I didn't realize that we were using openssl and sqlite.
Okay then. I've removed that line in 5e6de0c.

File: README.md (hunk 2 above)
Comment (LindirQuenya) [reply README.md]: Okay, sounds good.
Resolved in 15a4561.

**Review by XXLuigiMario — APPROVED** [review]
